.github/workflows/dockerbuild.yaml

```diff
@@ -122,3 +122,39 @@ jobs:
         repository: ibm-cloud-architecture/refarch-kc-gitops
         event-type: gitops-refresh
         client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}", "source": "${{ github.repository }}"}'
+
+  update-gitops:
+    needs:
+      build-docker-images
+    runs-on: ubuntu-latest
+    steps:
+    - name: Configure git client
+      id: configure-git
+      uses: djones6/git-config-action@master
+      with:
+        user-email: ${{ secrets.GITOPS_EMAIL }}
+        gitops-token: ${{ secrets.GITOPS_TOKEN }}
+    - name: Retrieve fleet-ms app-deploy.yaml
+      uses: actions/download-artifact@v2
+      with:
+        name: fleetms-app-deploy.yaml
+        path: fleetms
+    - name: Update fleet-ms app-deploy.yaml in GitOps repo
+      id: update-fleet-gitops
+      uses: djones6/appsody-gitops-update-action@master
+      with:
+        service-name: fleetms
+        github-org: ibm-cloud-architecture
+        gitops-repo-name: refarch-kc-gitops
+    - name: Retrieve voyages-ms app-deploy.yaml
+      uses: actions/download-artifact@v2
+      with:
+        name: voyagesms-app-deploy.yaml
+        path: voyagesms
+    - name: Update voyages-ms app-deploy.yaml in GitOps repo
```

Comment:
This all looks good but as the `appsody-gitops-update-action` is being externalized, I would add some logic like ` if [ "$(git status --porcelain | wc -l)" -gt "0" ]...` to make sure we only need to do `git add/commit` when something is there and there aren't benign workflow run failures. _(Similar to https://github.com/ibm-cloud-architecture/refarch-kc-gitops/blob/master/.github/workflows/update-gitops-deployments-(eda-integration).yaml#L41-L53)_